### ISSUE TITLE
Error `PING_ONE_DAVINCI is not a valid EnumProductType`

### ISSUE THREAD

**Issue by patrickcping** [pingidentity/terraform-provider-pingone#55]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### PingOne Terraform provider Version

v0.1.0

### Terraform Version

N/a

### Affected Resource(s)

- pingone_environment

### Terraform Configuration Files
```hcl
# Copy-paste your PingOne related Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.

terraform {
  required_providers {
    pingone = {
      source  = "pingidentity/pingone"
      version = "~> 0.1"
    }
  }
}

provider "pingone" {
  client_id      = var.client_id
  client_secret  = var.client_secret
  environment_id = var.environment_id
  region         = var.region

  force_delete_production_type = false
}

resource "pingone_environment" "my_environment" {
  name        = var.release_env_name
  description = "Created by Terraform"
  type        = "SANDBOX"
  license_id  = var.license_id

  default_population {
    name        = "My Population"
    description = "My new population for users"
  }

  service {
    type = "SSO"
  }

  service {
    type        = "PingFederate"
    console_url = "https://my-pingfederate-console.example.com/pingfederate"
  }
}
```

### Debug Output

```
Error: Error when calling `EnvironmentsApi.CreateEnvironmentActiveLicense``: PING_ONE_DAVINCI is not a valid EnumProductType
│ 
│   with pingone_environment.my_environment,
│   on main.tf line 19, in resource "pingone_environment" "my_environment":
│   19: resource "pingone_environment" "my_environment" {
│ 
│ Full HTTP response: {}
```

### Steps to Reproduce

1. `terraform apply`

**Comment by patrickcping** [comment]
Downstream SDK problem

**Comment by patrickcping** [comment]
Fixed in provider version v0.1.1

```hcl
terraform {
  required_providers {
    pingone = {
      source  = "pingidentity/pingone"
      version = "0.1.1"
    }
  }
}
....
```

`terraform init -upgrade`
